### PR TITLE
[CDAP-18038] Empty and error messaging to match app standard

### PR DESCRIPTION
**Pull request by njbriggs**
Switched to use EmptyMessageContainer and ErrorBanner

Jira: https://cdap.atlassian.net/browse/CDAP-18038

Empty state:
image

Error state:
image

### CODE REVIEW

File: app/cdap/components/Connections/Browser/GenericBrowser/index.tsx [hunk 1]
```diff
@@ -141,15 +145,23 @@ export function GenericBrowser({ selectedConnection }) {
...
-      <If condition={isEmpty && !error}>
-        <MuiAlert severity="info" className={classes.alert}>
-          No entities available
-        </MuiAlert>
+      <If condition={isEmpty && !loading}>
+        <EmptyMessageContainer title="No entities available">
+          <ul>
+            <li>
+              <span className="link-text" onClick={clearSearchString}>
+                {T.translate(`features.EmptyMessageContainer.clearLabel`)}
```
Comment (lahwran) [inline app/cdap/components/Connections/Browser/GenericBrowser/index.tsx:152]: What makes the difference between this string, which gets translated, and the strings on lines 155 and 158, which currently do not? Is this intentional?

File: app/cdap/components/Connections/Browser/GenericBrowser/index.tsx [hunk 2]
```diff
@@ -141,15 +145,23 @@ export function GenericBrowser({ selectedConnection }) {
           loading={loading}
         />
       </If>
-      <If condition={isEmpty && !error}>
-        <MuiAlert severity="info" className={classes.alert}>
-          No entities available
-        </MuiAlert>
+      <If condition={isEmpty && !loading}>
```
Comment (lahwran) [inline app/cdap/components/Connections/Browser/GenericBrowser/index.tsx:147]: Should this be visible in the event of an error which is shown in the ErrorBanner component below, on line 164?

Comment (njbriggs) [reply app/cdap/components/Connections/Browser/GenericBrowser/index.tsx:147]: Yes, this matches the current behavior.

File: app/cdap/components/Connections/Browser/GenericBrowser/index.tsx (hunk 1 above)
Comment (njbriggs) [reply app/cdap/components/Connections/Browser/GenericBrowser/index.tsx:152]: This should all be internationalized. Good catch.

Comment (njbriggs) [reply app/cdap/components/Connections/Browser/GenericBrowser/index.tsx:152]: Updated with full I18n for this portion.

**Review by ajainarayanan — APPROVED** [review]

**Pull request merged**